### DISCUSSION TITLE
compattamento

### DISCUSSION THREAD

**Discussion by marcolino04** [Courseplay/Courseplay_FS22#2687] [General]
hello we should improve the compaction in the open trenches[](url)[](url)

**Comment by marcolino04** [comment]
![Uploading Screenshot (100).png…]()
![Uploading Screenshot (99).png…]()

**Comment by marcolino04** [comment]
coursplay alone does not recognize the compacting tool

**Comment by Tensuko** [comment]
Pls open a correct issue with the template filled in and when you upload a screenshot, it need some patience until its done.
